plugins/ltac2/tac2entries.mli

```diff
@@ -45,7 +45,14 @@ val parse_scope : sexpr -> scope_rule
 
 (** {5 Inspecting} *)
 
-val print_ltac : Libnames.qualid -> unit
+val print_located_tactic : Libnames.qualid -> unit
+(** Display the absolute name of a tactic. *)
```

Comment:
Does this mean the fully qualified name?

Reply:
I copied this comment from tacentries.mli (ltac1), it might not be fully accurate